src/components/10-atoms/checkbox/index.js

```diff
@@ -168,6 +177,18 @@ class AXACheckbox extends NoShadowDOM {
       this.querySelector('input').checked = true;
       this.state.native = true;
     }
+
+    const iconElement = this.querySelector('.a-checkbox__icon');
+    const labelElement = document.querySelector(`label[for="${this.refId}"`);
+    const isSibling = labelElement
+      ? !!(labelElement.previousSibling || labelElement.nextSibling)
+      : false;
```

Comment:
`labelElement && !!(labelElement.previousSibling || labelElement.nextSibling))`

Should be easier, right?

Reply:
Yes makes sense, thx

Reply:
But my version would be a bit easier to understand, in my opinion